src/ipinfoWrapper.ts

```diff
@@ -69,6 +71,14 @@ export default class IPinfoWrapper {
      * @return Response containing location information.
      */
     public lookupIp(ip: string): Promise<IPinfo> {
+        if (this.isBogon(ip)) {
+            const ipinfo: IPinfo = {} as IPinfo;
+            ipinfo.bogon = true;
+            ipinfo.ip = ip;
+            return new Promise((resolve) => {
+                resolve(ipinfo);
+            });
+        }
```

Comment:
Code separation fix.

```suggestion
        }

```